index.html

```diff
@@ -54,6 +54,38 @@ <h4 id="starredHeader" class="hidden">Starred</h4>
     <p>Brought to you by the Network. <a href="https://github.com/xmatthewx/network-pulse">Contribute</a> to this prototype.</p>
   </footer>
 
+  <!-- Project Card template -->
+  <script type="text/template" id="project-card">
+    <%
+      var classNames = [
+        "project",
+        typeof(colorName)!== 'undefined' ? colorName : "",
```

Comment:
had to do`typeof` here to check the `var`... simply checking `colorName` would break `lodash`